docs/content/docs/try-flink/local_installation.md

```diff
@@ -141,7 +141,7 @@ You can view the data flow plan for the execution:
 
 Here for the job execution, Flink has two operators. The ﬁrst is the source operator which reads data from the
 collection source. The second operator is the transformation operator which aggregates counts of words. Learn
-more about [DataStream operators]({{< ref "docs/dev/datastream/operators" >}}).
+more about [DataStream operators]({{< ref "docs/dev/datastream/operators/overview" >}}).
```

Comment:
seems to be fixed in [https://nightlies.apache.org/flink/flink-docs-release-2.0-preview1/docs/try-flink/local_installation/](https://nightlies.apache.org/flink/flink-docs-release-2.0-preview1/docs/try-flink/local_installation/)

but is missing from master